### PR TITLE
update readme.txt

### PR DESCRIPTION
**Pull request by InpsydeNiklas**
updates the readme.txt

### CODE REVIEW

File: readme.txt [hunk 1]
```diff
@@ -8,48 +8,139 @@ Stable tag: 2.2.2
...
 
-PayPal's latest payments processing solution. Accept PayPal, Pay Later, credit/debit cards, alternative digital wallets and bank accounts.
+PayPal's latest payment processing solution. Accept PayPal, Pay Later, credit/debit cards, alternative digital wallets and bank accounts.
 
 == Description ==
 
-PayPal's latest, most complete payment processing solution. Accept PayPal exclusives, credit/debit cards and local payment methods. Turn on only PayPal options or process a full suite of payment methods. Enable global transactions with extensive currency and country coverage.
-Built and supported by [WooCommerce](https://woocommerce.com) and [PayPal](https://paypal.com).
+One checkout solution. Many ways to pay. PayPal’s all-in-one solution allows you to offer PayPal, Venmo (US), Pay Later at no additional cost, credit and debit cards, and country-specific payment options.
+= The all-in-one checkout solution* =
```
Comment (Dinamiko) [inline readme.txt:16]: Is asterisk intented here? `solution*`

File: readme.txt [hunk 2]
```diff
@@ -8,48 +8,139 @@ Stable tag: 2.2.2
...
+  
+- **Fraud Detection**: Save time and money by letting PayPal help you handle the risk of fraudulent transactions with our fraud, chargeback, and Seller Protection capabilities (on eligible transactions†). Our AI technology works hard to monitor all of your transactions — so you can focus on what matters most.†Available on eligible purchases. Limits apply.
+  
+- **Global compliance**: PayPal payment solutions help you meet your global compliance standards, such as PCI and PSD2, bringing international markets within reach for your business. Our platform has built-in compliance with two-factor authentication provided by cards (3D Secure). We include automatic updates as security protocols change.
+
+---
+
+**Legal Disclosures:**
+
+\*For Australian users, the PayPal service is provided by PayPal Australia Pty Limited AFSL 304962. Any information provided is general only and does not take into account your objectives, financial situation or needs. Please read and consider the CFSGPDS (paypal.com.au) before acquiring or using the service. See website for TMD.
```
Comment (Dinamiko) [inline readme.txt:71]: Seems `\*` is not needed here,  can you please confirm?

File: readme.txt [hunk 3]
```diff
@@ -79,6 +177,7 @@ Follow the steps below to connect the plugin to your PayPal account:
 5. Click "Connect to PayPal" to link your site to your PayPal account.
 6. Main settings screen.
 
+
```
Comment (Dinamiko) [inline readme.txt:180]: Seems this new line is not needed here, doesn't it?

**Review by Dinamiko — CHANGES_REQUESTED** [review]

File: readme.txt (hunk 1 above)
Comment (InpsydeNiklas) [reply readme.txt:16]: This relates to the legal disclosures below.

File: readme.txt (hunk 2 above)
Comment (InpsydeNiklas) [reply readme.txt:71]: This `\` is needed to nullify the `*` which relates to the above.

File: readme.txt (hunk 3 above)
Comment (InpsydeNiklas) [reply readme.txt:180]: This section is subject to change, so it should be fine for now.

**Review by Dinamiko — APPROVED** [review]

**Pull request merged**